src/tides.f90

```diff
@@ -20,36 +22,49 @@
 !!
 !> @anchor      tides
 !!
-!> @todo        Find bug in Solid tides (comparing to 2-body trajectory -> discontinuity?!)
-!> @todo        Ocean tide model implementation incorrect? No reference and Vallado seems to be wrong. Consider implementing a REAL IERS model
 !!------------------------------------------------------------------------------------------------
 module tides
 
-  use slam_astro,             only: getEarthRadius, getEarthGravity, getEarthMass
+  use slam_astro,             only: getEarthRadius, getEarthGravity, getEarthMass, getGMST
   use slam_astro_conversions, only: getGeodeticLatLon, getRadiusLatLon
   use neptune_error_handling, only: E_SOLAR_SYSTEM_INIT, setNeptuneError
   use slam_error_handling,    only: isControlled, hasToReturn, hasFailed, FATAL, checkIn, checkOut
-  use slam_math,              only: mag, pi, eps9, rad2deg, factorial
+  use slam_math,              only: mag, pi, eps9, rad2deg, factorial, deg2rad, as2deg
   use slam_reduction_class,   only: Reduction_type
   use solarsystem,            only: Solarsystem_class, ID_SUN, ID_MOON
   use slam_types,             only: dp
+  use slam_io,                only: openFile, closeFile, SEQUENTIAL, IN_FORMATTED
+  use slam_time,              only: jd2000, jd245, julian_century
 
   implicit none
 
   private
 
-    integer, parameter, public :: SOLID_TIDES = 1
-    integer, parameter, public :: OCEAN_TIDES = 2
+    integer, parameter, public          :: SOLID_TIDES = 1
+    integer, parameter, public          :: OCEAN_TIDES = 2
+    integer, parameter, public          :: max_l = 6
+    character(len=*), parameter, public :: path = "../work/data/fes2004_Cnm-Snm.dat"
```

Comment:
This path is hard coded, which  should be replaced by a variable "path" that is determined at runtime.This is also done in atmosphere.f90: 

- A path is handed over to the init function as a parameter
- Then the data_path is constructed
- With this the path to the respective file is constucted

The reason to make this so dynamic, is that NEPTUNE is also called via API. And in that setup there is no 'work' folder but the data files can be located anywhere else.

Reply:
@andturchi: one more thing is left. Look into atmosphere.f90, the init subroutine takes the cpath argument. It is then called by NEPTUNE with a path. Can you adept the tides.f90 accordingly?